app/views/playgrounds/show.html.erb

```diff
@@ -10,7 +10,7 @@
   <%= participation.user.nickname  %>
   <%= participation.start_date  %>
   <%= participation.end_date  %>
-  <%= participation.status  %>
+  <%= participation.define_status  %>
```

Comment:
hey man you should rather call it "status" right ?

Reply:
Need also to change it in the model.